ios/Classes/BetterPlayer.m

```diff
@@ -113,10 +119,25 @@ - (void) removeObservers{
                                    forKeyPath:@"playbackBufferFull"
                                       context:playbackBufferFullContext];
         [[NSNotificationCenter defaultCenter] removeObserver:self];
+        
+        if (_timeObserverId) {
+            [_player removeTimeObserver:_timeObserverId];
+            _timeObserverId = nil;
+        }
         self._observersAdded = false;
     }
 }
 
+- (void)notifyPlaybackChangeInPiP {
+    int64_t position = [self position];
+    
+    if (_isPipMode && !_isLiveStream && position >= 0) {
```

Comment:
The event name (`playbackStatusChangeInPiP`) doesn't have "is Live?" meaning.
How about checking it on the app-side?

Reply:
I made the change here [44e1207](https://github.com/dwango-nfc/betterplayer/pull/54/commits/44e120704d1928cbf9f7f936ee9ff049d2014bba),[7680afa](https://github.com/dwango-nfc/dwango-app-ch/pull/1811/commits/7680afa8ce68f07361c92f588e4328b0368f93c9)